scripts/hid_configurator/configurator.py

```diff
@@ -49,15 +51,71 @@
 DFU_SYNC_RETRIES = 3
 DFU_SYNC_INTERVAL = 1
 
-NORDIC_VID = 0x1915
-DEVICE_PID = {
-    'desktop_mouse_nrf52832'    : 0x52DA,
-    'desktop_mouse_nrf52810'    : 0x52DB,
-    'gaming_mouse'              : 0x52DE,
-    'keyboard'                  : 0x52DD,
-    'dongle'                    : 0x52DC,
+
+PMW3360_OPTIONS = {
+    'downshift_run':    ConfigOption((10,   2550),   SENSOR_OPT_DOWNSHIFT_RUN,   'Time in milliseconds of switching from mode Run to Rest 1.'),
+    'downshift_rest1':  ConfigOption((320,  81600),  SENSOR_OPT_DOWNSHIFT_REST1, 'Time in milliseconds of switching from mode Rest 1 to Rest 2.'),
```

Comment:
Maybe we could make description shorter - like: `Switching time: Run -> Rest 1 [ms]` (we got long lines here).
Same for second sensor options.

Reply:
Done